moped-editor/src/utils/materialTableHelpers.js

```diff
@@ -24,3 +26,25 @@ export const handleKeyEvent = e => {
     e.stopPropagation();
   }
 };
+
+export const useWindowResize = () => {
```

Comment:
@chiaberry The approach is good, but some commentary (docstrings) would be appreciated, anything that explains what this custom hook does.